lib/server.js

```diff
@@ -227,6 +227,10 @@ const contentTypeNegotiation = (req, res, next) => {
 app.use(contentTypeNegotiation)
 
 // FHIR root handler
+app.get('/fhir/metadata', (req, res, next) => {
+  req.url = '/DONT_MATCH/metadata' // Change url so that no more route matching is done else searches will clash
```

Comment:
It might be a bit less hacky to add the `outcomeHandler` middleware function to this same route after the current one. That way the outcome will be handled immediately and the router doesn't have to try and match the url against other routes.

Reply:
Thanks, that's much better.